Tests/JsonLDParserTest.swift

```diff
@@ -0,0 +1,122 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+import XCTest
+@testable import apio_consumer_ios
+
+/**
+ * @author Marcelo Mello
+ */
+class JsonLDParserTest : BaseTest {
+    
+    private var thing: Thing!
+    private var embeddedThings: Dict!
+    private var attributes: Attribute!
+    
+    override func setUp() {
+        super.setUp()
+        
+        let responseWithEmbedded = loadJson("response-item-with-embbeded-structure")
+        let (thing, embeddedThings) = JsonLDParser.parseThing(json: responseWithEmbedded)
+        
+        self.thing = thing
+        self.embeddedThings = embeddedThings
+        self.attributes = thing.attributes
+    }
+    
+    func testShouldGetThingOperations() {
+        let operations = thing.operations
+        
+        var typesOperation = operations["_:form/evaluate-context"]?.types
+        XCTAssertEqual(operations["_:form/evaluate-context"]?.id, "_:form/evaluate-context")
```

Comment:
extract `operations["_:form/evaluate-context"]` in a variable, and all others that repeat too much as this.